### PR TITLE
Feat(chaos-stage)Adding chaos stage to coreDNS pipeline

### PR DESCRIPTION
**Pull request by rajdas98**
Signed-off-by: Raj <raj.das@mayadata.io>
Issue- https://github.com/crosscloudci/coredns-configuration/issues/41
- Adding the chaos stage for coreDNS test cases.
- Adding the coreDNS pod delete experiment using litmus to check the resiliency of the application

### CODE REVIEW

**Comment by umamukkara** [comment]
@denverwilliams would love to get your views on this PR.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -87,3 +87,44 @@ container:
...
+chaos:
+  stage: test
+  image: ubuntu
+  script:
+    - apt-get update && apt-get install docker.io -y && apt-get install curl -y && apt-get install wget
+    - service docker start
+    - docker version
+    - wget https://dl.google.com/go/go1.13.5.linux-amd64.tar.gz && tar zxf go1.13.5.linux-amd64.tar.gz -C /usr/local/
+    - export PATH=$PATH:/usr/local/go/bin
+    - GO111MODULE="on" go get sigs.k8s.io/kind@v0.6.1 && export PATH=$PATH:~/go/bin
```
Comment (taylor) [inline .gitlab-ci.yml]: Any reason for not using 0.7.0, @rajdas98?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -87,3 +87,44 @@ container:
...
+    - kind version
```
Comment (denverwilliams) [inline .gitlab-ci.yml]: This probably needs to be wrapped by a 
```
    - >
      if [ "$ARCH" == "amd64" ]; then
      <code>
      fi
```

File: .gitlab-ci.yml (hunk 1 above)
Comment (rajdas98) [reply .gitlab-ci.yml]: No reason @taylor . I upgraded to 0.7.0

File: .gitlab-ci.yml (hunk 2 above)
Comment (rajdas98) [reply .gitlab-ci.yml]: Hi @denverwilliams . KIND supported arm64. Here is the link https://github.com/kubernetes-sigs/kind/releases/tag/v0.7.0. 
I added the conditional logic to use arm64.

**Comment by umamukkara** [comment]
Thanks @denverwilliams for the review comments.

File: .gitlab-ci.yml (hunk 2 above)
Comment (taylor) [reply .gitlab-ci.yml]: lgtm

**Pull request merged**